pyparsing/util.py

```diff
@@ -275,10 +275,3 @@ def _inner(*args, **kwargs):
         _inner.__kwdefaults__ = None
     _inner.__qualname__ = fn.__qualname__
     return cast(C, _inner)
-
-
-def replaced_by_pep8(fn: C) -> Callable[[Callable], C]:
```

Comment:
Instead of deleting this and changing all the references to directly reference `_make_synonym_function`, please just rename `_make_synonym_function` to `replaced_by_pep8`

Reply:
Also, please change the commented line 263 to "stacklevel=2", so that I don't forget later when I uncomment this code.

Reply:
Thanks for the review, I should have addressed your comments in the last commit.